src/lxc/start.c

```diff
@@ -481,6 +481,7 @@ struct lxc_handler *lxc_init_handler(const char *name, struct lxc_conf *conf,
 	handler->conf = conf;
 	handler->lxcpath = lxcpath;
 	handler->pinfd = -1;
+	handler->state_socket = -1;
 	lxc_list_init(&handler->state_clients);
 
```

Comment:
It is better to move the changes in `start.c/start.h` to commit `start: use separate socket on daemonized start`